app/logic/users.py

```diff
@@ -47,6 +49,19 @@ def removeUserInterest(program_id, username):
         interestToDelete.delete_instance()
     return True
 
+def getBannedUsers(program):
+    """
+    This function returns users banned from a program.
+    """
+    return ProgramBan.select().where(ProgramBan.program == program)
+
+def isBannedFromEvent(username, eventId):
+    """
+    This function returns whether the user is banned from the program associated with an event.
+    """
+    program = ProgramEvent.get(ProgramEvent.event_id == eventId).program
```

Comment:
You don't necessarily have to, but we have an event property `singleProgram` that could shorten that line and make it a little simpler.

Reply:
Done